demo/Program.cs

```diff
@@ -23,26 +24,62 @@ static void Main(string[] args)
             var logger = serviceProvider.GetService<ILogger<Program>>();
             var monitor = serviceProvider.GetService<Monitor>();
 
-            monitor.IncomingPlayerEvent += (s, e) => {
+            monitor.IncomingPlayerEvent += (s, e) =>
+            {
                 logger.LogInformation($"INCOMING: {e.PlayerState}");
             };
-            monitor.OutgoingPlayerEvent += (s, e) => {
-                logger.LogInformation($"OUTGOING: {e.PlayerState}");
-            };
-            monitor.IncomingChatMessageEvent += (s, e) => {
-                logger.LogInformation($"CHAT: {e.Message}");
-            };
-            monitor.IncomingPlayerEnteredWorldEvent += (s, e) => {
-                logger.LogInformation($"WORLD: {e.PlayerUpdate}");
-            };
-            monitor.IncomingRideOnGivenEvent += (s, e) => {
-                logger.LogInformation($"RIDEON: {e.RideOn}");
-            };
+            //monitor.OutgoingPlayerEvent += (s, e) => {
+            //    logger.LogInformation($"OUTGOING: {e.PlayerState}");
+            //};
+            //monitor.IncomingChatMessageEvent += (s, e) => {
+            //    logger.LogInformation($"CHAT: {e.Message}");
+            //};
+            //monitor.IncomingPlayerEnteredWorldEvent += (s, e) => {
+            //    logger.LogInformation($"WORLD: {e.PlayerUpdate}");
+            //};
+            //monitor.IncomingRideOnGivenEvent += (s, e) => {
+            //    logger.LogInformation($"RIDEON: {e.RideOn}");
+            //};

```

Comment:
I'd prefer to not have sample code commented out like this. No harm in leaving it in the demo, no?

Reply:
Done.